docs/monitoring.md

```diff
@@ -273,8 +309,8 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Download the event logs for all attempts of the given application as a zip file</td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/[attempt-id]/logs</code></td>
-    <td>Download the event logs for the specified attempt of the given application as a zip file</td>
+    <td><code>/applications/[app-id]/logs</code></td>
+    <td>Download the event logs for the application as a zip file</td>
```

Comment:
ah good point.  So this case is a little different because both are supported and the one w/out `attemptId` is sometimes useful, to grab the logs for all attempts.  I thought the docs do actually spell this out, the first endpoint says it will download logs for all attempts.